cdp-agentkit-core/cdp_agentkit_core/actions/social/twitter/account_details.py

```diff
@@ -0,0 +1,33 @@
+
+import tweepy
+from pydantic import BaseModel
+
+ACCOUNT_DETAILS_PROMPT = """
+This tool will return account details for the authenticated user context."""
+
+class AccountDetailsInput(BaseModel):
```

Comment:
See https://github.com/coinbase/cdp-agentkit/blob/69438c3d82277a8160c37d6a38e41913a5cd3e1e/cdp-agentkit-core/cdp_agentkit_core/actions/get_wallet_details.py#L11

for an example of how we handle no input arg schemas.

Reply:
roger that!